articles/cosmos-db/mongodb/vcore/data-api.md

```diff
@@ -17,7 +17,24 @@ The Data API for Azure Cosmos DB for MongoDB vCore is an https interface that al
 
 ## Enabling Data API on MongoDB vCore
 
-You can enable or disable this feature using the Azure CLI or an ARM template. Portal support will be added soon.
+You can enable or disable this feature using the Azure Portal, Azure CLI or an ARM template.
+
+### Steps to enable Data API on Azure Portal
+
+1. **Go to Azure Portal**
+    - portal.azure.com
+
+2. **Go to your cluster**
```

Comment:
Go to your vCore-based Azure Cosmos DB for MongoDB cluster

Reply:
@cocobird1 Could you make these edits please?